ledger/participant-integration-api/src/main/resources/db/migration/postgres/V126__identity_provider_config.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE participant_identity_provider_config
+(
+    identity_provider_id VARCHAR(255) PRIMARY KEY NOT NULL,
+    issuer               VARCHAR             NOT NULL UNIQUE,
+    jwks_url             VARCHAR             NOT NULL,
+    is_deactivated       BOOLEAN             NOT NULL
```

Comment:
Consider adding ` created_at          NUMBER          NOT NULL`.
It would be unused the same as user's created_at but it might useful when debugging an issue in a live system